### PR TITLE
Release-notes-first-cut-6-19

### PR DESCRIPTION
**Pull request by kaarthis**

### CODE REVIEW

**Review by qpetraroia — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,34 @@
...
+  * CoreDNS version 1.9.2 will be default version. With this new version of CoreDNS wildcard queries are no longer allowed.
+  * metrics-server version 0.6.1 will be the default version.
+* Konnectivity rollout will continue in May 2022 and is expected to complete by end of June.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+
+### Release notes
+
+* Behavioral Changes
+  * Some of the KEDA scalers will no longer be able to scale workloads. When describing the HPA created for the KEDA scaled object through a command like kubectl describe hpa <hpa-name>, you   may see a warning message such as - Cannot list resource "<external-metric-name>" in API group "external.metrics.k8s.io " in the namespace "<namespace-name>": RBAC: clusterrole.rbac.authorization.k8s.io  "keda-operator-external-metrics-reader" not found
```
Comment (justindavies) [inline CHANGELOG.md:21]: Are there docs to point customers at?

Comment (shashankbarsin) [inline CHANGELOG.md:21]: on 'some of the KEDA scalers' - which scalers? do we have a list of scalers?

Also, any doc we can link to?

Comment (kaarthis) [reply CHANGELOG.md:21]: Good point similarly to what Justin Mentioned. Yes we have list of scalers documented link to https://keda.sh/docs/2.7/scalers/

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,34 @@
...
+  * PVC mounts will now work in clouds with custom root CAs.
+  * Nodepool snapshots will only allow taking snapshots from Nodepools with provisioning status as Succeeded.
+  * Added node restriction property for api model, which will allow user to enable/disable node restriction.
```
Comment (shashankbarsin) [inline CHANGELOG.md:24]: suggest changing api model -> AKS resource API. Any doc we can link to? Also, should this be under behaivoral changes (the default will match current behavior?) or under announcements?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,34 @@
...
+### Release notes
+
+* Behavioral Changes
+  * Some of the KEDA scalers will no longer be able to scale workloads. When describing the HPA created for the KEDA scaled object through a command like kubectl describe hpa <hpa-name>, you   may see a warning message such as - Cannot list resource "<external-metric-name>" in API group "external.metrics.k8s.io " in the namespace "<namespace-name>": RBAC: clusterrole.rbac.authorization.k8s.io  "keda-operator-external-metrics-reader" not found
+  * PVC mounts will now work in clouds with custom root CAs.
+  * Nodepool snapshots will only allow taking snapshots from Nodepools with provisioning status as Succeeded.
+  * Added node restriction property for api model, which will allow user to enable/disable node restriction.  
+* Component Updates
+  * AKS Ubuntu 18.04 image updated to [AKSUbuntu-1804-2022.06.13](vhd-notes/aks-ubuntu/AKSUbuntu-1804/2022.06.13.txt).
+  * Upgrade Azure File CSI driver to [v1.19.0](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.19.0)
```
Comment (shashankbarsin) [inline CHANGELOG.md:27]: repeat from previous release notes. do we have a version > 1.19.0 for this release? if not, we can remove this

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,34 @@
...
+* Starting with Kubernetes 1.24, the following changes will be made:
+  * The default format of clusterUser credential for AAD enabled clusters will be ‘exec’, which requires [kubelogin](https://github.com/Azure/kubelogin) binary in the execution PATH. If you are using Azure CLI, it will prompt users to download kubelogin. There will be no behavior change for non-AAD clusters, or AAD clusters whose version is older than 1.24. Existing downloaded kubeconfig will still work. We provide an optional query parameter ‘format’ when getting clusterUser credential to overwrite the default behavior change, you can explicitly specify format to ‘azure’ to get old format kubeconfig.
+  * The [NodeRestriction Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction) will be enabled
+  * CoreDNS version 1.9.2 will be default version. With this new version of CoreDNS wildcard queries are no longer allowed.
+  * metrics-server version 0.6.1 will be the default version.
```
Comment (shashankbarsin) [inline CHANGELOG.md:14]: these two are repeated in previous 2 release notes. do we still need them?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,34 @@
...
+  * PVC mounts will now work in clouds with custom root CAs.
+  * Nodepool snapshots will only allow taking snapshots from Nodepools with provisioning status as Succeeded.
+  * Added node restriction property for api model, which will allow user to enable/disable node restriction.  
+* Component Updates
+  * AKS Ubuntu 18.04 image updated to [AKSUbuntu-1804-2022.06.13](vhd-notes/aks-ubuntu/AKSUbuntu-1804/2022.06.13.txt).
+  * Upgrade Azure File CSI driver to [v1.19.0](https://github.com/kubernetes-sigs/azurefile-csi-driver/releases/tag/v1.19.0)
+  * AKS Windows 2019 image has been updated to [17763.2928.220615](vhd-notes/AKSWindows/2019/17763.3046.220615.txt).
+  * AKS Windows 2022 image has been added with version [20348.707.220525](vhd-notes/AKSWindows/2022/20348.768.220615.txt).
+  * Upgraded cloud-controller-manager network API version to 2021-08-01
```
Comment (shashankbarsin) [inline CHANGELOG.md:30]: is this user facing API?

File: CHANGELOG.md (hunk 4 above)
Comment (kaarthis) [reply CHANGELOG.md:14]: I thought there maybe value but happy to remove it.

File: CHANGELOG.md (hunk 5 above)
Comment (kaarthis) [reply CHANGELOG.md:31]: it was marked customer facing in the PR so i assume so but let me validate.

File: CHANGELOG.md (hunk 1 above)
Comment (kaarthis) [reply CHANGELOG.md]: No docs just a PR unfortunately.

Comment (kaarthis) [reply CHANGELOG.md]: All the scalers. It was a cluster role missing issue that got fixed : Fixed an issue related to missing KEDA Cluster-role : Any of the KEDA scalers may have noticed the following issue when scaling workloads,i.e When describing the HPA created for the KEDA scaled object through a command like kubectl describe hpa <hpa-name>, you  may now see a warning message such as - Cannot list resource "<external-metric-name>" in API group "external.metrics.k8s.io " in the namespace "<namespace-name>": RBAC: clusterrole.rbac.authorization.k8s.io  "keda-operator-external-metrics-reader" not found. The issue is now fixed by adding the missing cluster role - clusterrole.rbac.authorization.k8s.io.

File: CHANGELOG.md (hunk 2 above)
Comment (kaarthis) [reply CHANGELOG.md]: Yes modified the announcement with one additional point .

File: CHANGELOG.md (hunk 5 above)
Comment (kaarthis) [reply CHANGELOG.md]: Removed as it was not customer facing.

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,34 @@
...
+  * The [NodeRestriction Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction) will be enabled.  This will allow users to enable/disable node restriction.
+  * CoreDNS version 1.9.2 will be default version. With this new version of CoreDNS wildcard queries are no longer allowed.
+  * metrics-server version 0.6.1 will be the default version.
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022. 
+* Konnectivity rollout will continue in May 2022 and is expected to complete by end of June.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+
+### Release notes
+* Features
```
Comment (palma21) [inline CHANGELOG.md:20]: Preview Features

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,34 @@
...
+  * The [NodeRestriction Admission Controller](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#noderestriction) will be enabled.  This will allow users to enable/disable node restriction.
+  * CoreDNS version 1.9.2 will be default version. With this new version of CoreDNS wildcard queries are no longer allowed.
+  * metrics-server version 0.6.1 will be the default version.
+* Kubernetes 1.21 version deprecation will start taking effect from July 31st, 2022. 
+* Konnectivity rollout will continue in May 2022 and is expected to complete by end of June.
+* Update your AKS labels to the recommended substitutions before deprecation after the Kubernetes v1.24 release. See more information on label deprecations and how to update your labels in the [Use labels in an AKS cluster](https://docs.microsoft.com/azure/aks/use-labels) documentation.
+
+### Release notes
+* Preview Features
+  * Disable [CSI Storage Drivers](https://docs.microsoft.com/en-us/azure/aks/azure-disk-csi) available in Preview.
```
Comment (shashankbarsin) [inline CHANGELOG.md:21]:
```suggestion
  * Disable [CSI Storage Drivers](https://docs.microsoft.com/azure/aks/azure-disk-csi) available in preview.
```

**Review by shashankbarsin — APPROVED** [review]

**Pull request merged**